src/config/docker.md

```diff
@@ -0,0 +1,96 @@
+# Docker
+
+Docker is a platform of products to build isolated containers for programs using
+visualization.
+
+## Getting Started with Docker.
+
+After installing the 'docker' package (and optionally, the `docker-compose`
```

Comment:
This is a nitpick, but I'd really prefer if this said just 'install the `docker` package and enable the docker service.

I'm also not sure we need to advertise the `docker-compose` package. It's package name is very intuitive.